docs/debugging.md

```diff
@@ -123,15 +123,14 @@ The easiest way to test the pack functionality with dotnet.exe is to install the
 
 If you want to test dotnet.exe explicitly, so you don't have to worry about whether you installed the correct package in the project, refer to [Patching dotnet.exe to test the NuGet functionality](#patching-dotnetexe-to-test-the-nuget-functionality).
 
-### Debugging NuGet Command Xplat Functionality (add-package/remove-package/list package)
-
-Functionality such as `dotnet.exe add package` or `dotnet list package`, is implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
-
-To debug with `MSBuildLocator`, you need to un-comment the `PackageReference` for `Microsoft.Build.Locator` in `NuGet.CommandLine.XPlat.csproj`. Additionally, un-comment the code utilizing `MSBuildLocator` in that project's `Program.cs`.
+### Debugging NuGet's integrated `dotnet` CLI commands
+All of the `dotnet nuget` commands, in addition to some others, such as `dotnet add package`, `dotnet list package`, and `dotnet search`, are implemented in [NuGet.CommandLine.XPlat](../src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj).
```

Comment:
I'm pretty sure that if you install a markdown linter, it'll suggest to you to have a line of whitespace both before and after headings.